worker-message-prioritization-distribution/pom.xml

```diff
@@ -141,7 +141,7 @@
                         <CAF_RABBITMQ_PASSWORD>guest</CAF_RABBITMQ_PASSWORD>
                         <CAF_WMP_KUBERNETES_NAMESPACES>private</CAF_WMP_KUBERNETES_NAMESPACES>
                         <CAF_LOG_LEVEL>INFO</CAF_LOG_LEVEL>
-                        <CAF_NOOP_MODE>true</CAF_NOOP_MODE>
+                        <CAF_ENABLE_TARGET_QUEUE_LENGTH_TUNING>true</CAF_ENABLE_TARGET_QUEUE_LENGTH_TUNING>
```

Comment:
Should this have been false?

Reply:
Because of this comment I have put it on by default 
![image](https://github.com/WorkerFramework/worker-message-prioritization/assets/59800744/1c397998-e85e-4233-aeaa-014cbebca6cf)

Reply:
I misunderstood you, this has now been set to false here, and to true in deploy